client/src/components/new-request/name-input.vue

```diff
@@ -6,8 +6,10 @@
                   autocomplete="off"
                   filled
                   id="name-input-text-field"
+                  :rules="isMrasSearch ? mrasRules : []"
+                  ref="ifCorpNumInput"
```

Comment:
I don't get the name here. Shouldn't it be close to the element id?

Reply:
For sure, will organize this better!